lighthouse-core/gather/computed/critical-request-chains.js

```diff
@@ -67,7 +71,8 @@ class CriticalRequestChains extends ComputedArtifact {
         startTime: request.startTime,
         endTime: request.endTime,
         responseReceivedTime: request.responseReceivedTime,
-        transferSize: request.transferSize
+        transferSize: request.transferSize,
+        blocked: request.statusCode === 0
```

Comment:
not sure if there is a better way to tell whether a request is blocked by dev-tool. I will use JavaScript to find all the blocked requests instead if necessary.